cluster/ubuntu/skydns-rc.yaml.template

```diff
@@ -1,7 +1,7 @@
 kind: ReplicationController
 apiVersion: v1beta1
 id: kube-dns
-namespace: default
+namespace: kube-system
```

Comment:
This doesn't do any harm, but these v1beta1 yaml files are completely broken at this point.